conda-recipe/run_test.bat

```diff
@@ -1,4 +1,4 @@
-@echo off
+@echo on
```

Comment:
Can you please attach the log examples "before" and "after" to this PR.

Reply:
It doesn't add any extra prints to the logs produced here. I _think_ it requires the caller script to not have supressed echo either.

When it works, this `@echo on` should  print the commands that it will execute before it executes them, but after variable substitutions, which can be helpful when testing things outside of the CI pipelines.

Reply:
The comment is related not to this line, but to the whole PR. Sorry for the misunderstanding.
The description says that it fixes the clashes in the JSON repost in case of distributed execution.
So, please attach the reports "before" and "after" the changes.

Reply:
I unfortunately don't have report files to attach.

But the issue is quite simple: multiple processes were writing to the same output file at the same time, which results in malformed json - as a simple example, if two processes both write "out" at the same time to the same file, then the output might be one of "outout", "oouutt", "ououtt", "ooutut", etc., which in the case of JSON text results in a file which doesn't even follow json format as `{}`, `:`, `[]` and so on are not respected.